test/Benchmarks/src/Startup/Import_World.enso

```diff
@@ -0,0 +1,9 @@
+from Standard.Base import all
```

Comment:
This example was obtained by running `enso --new HelloExample` and then the body of `main` was changed.

Reply:
The second [stdlib benchmarks run](https://github.com/enso-org/enso/actions/runs/10596279560) has improved all the startup benchmarks, but only minimally by 4%%, 1%% and 17%% - that's below the measurement precision we have. I was expecting bigger speedup according to the VisualVM graphs.

![Expected 700ms](https://github.com/user-attachments/assets/97937aa4-c8aa-4379-b6c0-74f5e9580a2f)

Reply:
The new version 0fba66b removes any trace of `registerPolyglotSymbol` from the (10ms interval) sampling. That's a good sign. However there still seems to be `getPolyglotSymbol` caught in the trace two times:

![getPolyglotSymbol](https://github.com/user-attachments/assets/2ac41a3f-d0ac-47d4-a401-aeb5776a848d)

Maybe we could improving the startup time by using `Supplier<Object>` where the _polyglot symbol_ is requested by the `IrToTruffle`.